tests/phpunit/tests/user/wpRegisterPersistedPreferencesMeta.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * @group user
+ *
+ * @covers ::wp_register_persisted_preferences_meta
+ */
+class Tests_User_wpRegisterPersistedPreferencesMeta extends WP_UnitTestCase {
+
+	/**
+	 * Test that user persisted preferences meta is registered.
+	 *
+	 * @ticket 56467
+	 */
+	public function test_should_register_persisted_preferences_meta() {
+		global $wpdb, $wp_meta_keys;
+		$meta_key = $wpdb->get_blog_prefix() . 'persisted_preferences';
+
+		// Test that meta key is registered.
+		unregister_meta_key( 'user', $meta_key );
+		wp_register_persisted_preferences_meta();
+
+		$this->assertIsArray( $wp_meta_keys, 'No meta keys exist' );
+		$this->assertArrayHasKey(
+			$meta_key,
+			$wp_meta_keys['user'][''],
+			'The expected meta key was not registered'
+		);
+
+		// Test to detect changes in meta key structure.
+		$structure = $wp_meta_keys['user'][''][ $meta_key ];
```

Comment:
Hmm, on a second look, should we change this variable name to `$actual` for consistency and clarity?

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/3253/commits/0af827e7c4a11c6be944091d36c18c46201f7a93 (though I removed the variable for consistency ✅